pymysql/connections.py

```diff
@@ -534,7 +534,8 @@ def __init__(self, host=None, user=None, password="",
                  compress=None, named_pipe=None, no_delay=None,
                  autocommit=False, db=None, passwd=None, local_infile=False,
                  max_allowed_packet=16*1024*1024, defer_connect=False,
-                 auth_plugin_map={}, read_timeout=None, write_timeout=None):
+                 auth_plugin_map={}, read_timeout=None, write_timeout=None,
+                 bind=None):
```

Comment:
I prefer bind_address rather than bind. (Same to option name, translated '-' with '_')